api/LinkyLink.sln

```diff
@@ -1,56 +1,31 @@
 ﻿
```

Comment:
What are our best practices regarding sln files? They could differ from one dev to another and thus they might become a part of every commit. Can we generalize it so that its the same and only differs if projects were added/removed?

Reply:
Agreed, this should only be changed on adding or updating the projects.